core/domain/exp_jobs_one_off.py

```diff
@@ -540,3 +540,18 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class DeleteStateIdMappingModelsOneOffJobs(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Do we want this to be inside job_registry.py?

Reply:
Added it there. Good catch!